scheduler/src/cook/components.clj

```diff
@@ -319,6 +323,10 @@
   (println "Cook" @util/version "( commit" @util/commit ")")
   (try
     (mount/start-with-args (cook.config/read-config config-file-path))
+    ; Note: If the mount/start-with-args fails to initialize a defstate S, and/or you get weird errors on startup,
+    ; you need to require S's namespace with ns :require. 'ns :require' is how mount finds defstates to initialize.
+    ; Note, we cannot directly require cook.mesos.api. This namespace is normally hidden from mount because cook.mesos.api is
+    ; loaded via util/lazy-load-var.
```

Comment:
The top 2 lines of this comment make sense to me, but maybe we should put them above the call to `mount/start-with-args`?

I don't understand why the bottom 2 lines of the comment are needed here.